### ISSUE TITLE
Update execution workflow output file specifications to report fractional relative usage values

### ISSUE THREAD

**Issue by SamBryce-Smith** [iRNA-COSI/APAeval#372]
As in title, we want to evaluate the relative usage values reported by tools and we need to capture them into standardised output formats for consumption by summary workflows.

Specifications (`execution_workflows/execution_output_specification.md`) need to be updated before we can modify the execution workflows.

**Comment by SamBryce-Smith** [comment]
After discussion in meeting 13/07 this issue/linked PR now focuses on reporting **fractional polyA site usage**. Separate issue will be created for deciding how to output tool-specific 'scores' for regions (genes/terminal exons).
